fuse_optimizers/test/launch_tests/test_fixed_lag_ignition.cpp

```diff
@@ -131,6 +131,11 @@ TEST_F(FixedLagIgnitionFixture, SetInitialState)
   pose_msg1.pose.covariance[35] = 1.0;
   relative_pose_publisher->publish(pose_msg1);
 
+  /// @todo(swilliams) There is a timing or message queuing issue. This sleep "solves"
```

Comment:
This unrelated unit test was failing on my machine. It appears to be a timing or message queue issue. The Pose2D sensor is only receiving the second published messages with a timestamp of 3.0s. The previous message with a timestamp of 2.0s is never processed. As a quick fix, I just added a delay. But this seems like a problem that should be fixable by changing a message queue size somewhere.